mopidy/core/library.py

```diff
@@ -273,6 +296,12 @@ def search(self, query=None, uris=None, exact=False, **kwargs):
                 logger.warning(
                     '%s does not implement library.search() with "exact" '
                     'support. Please upgrade it.', backend_name)
+            except LookupError:
+                raise
```

Comment:
I breaks some test if I don't let it through, possibly one the ones that checks for bad fields or something like that. Wasn't sure about changing it so here we are. If nothing else I can add a comment about what this is enabling and we can postpone changing the rest of it.